Automattic/wp-calypso
# Pro plan: add missing feature related to the site migration

#### Changes proposed in this Pull Request

It fixes the case when a user tries to migrate a site to the site with the Pro plan without redirecting to the upgrade screen since it has a proper plan.

#### Testing instructions

- Go to `/migrate/{SLUG}
- Enter the source site (Jetpack connection should be established)
- Select Everything options
- Check if there is a proper step without an upgrade plan screen

#### Screenshots

**Before**
<img width="737" alt="Screenshot 2022-04-05 at 18 42 56" src="https://user-images.githubusercontent.com/1241413/161804418-9e8f2f75-d5ee-4891-a60f-5769331160c7.png">

**After**
<img width="741" alt="Screenshot 2022-04-05 at 18 42 12" src="https://user-images.githubusercontent.com/1241413/161804446-89d005c9-c1fd-4ee3-b645-0ce8500a3e3d.png">
